iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -976,6 +976,35 @@ def align_index(self, constraints: list["Constraint"]) -> None:
         if is_shared_mem_access(self):
             self.index = align_index_vars(self.index, constraints)
 
+    def has_identity_mapping(self) -> bool:
+        mapping = self.mapping
+        if mapping is None:
+            return True
+
+        mem_shape = self.memory.type.symbolic_shape
+        if mapping.is_identity() and mapping.input_shape == mem_shape:
+            return True
+
+        return False
```

Comment:
How about
```
def has_identity_mapping(self) -> bool:
        return self.mapping is None or (
            self.mapping.is_identity() and self.mapping.input_shape == self.memory.type.symbolic_shape
        )
```

Reply:
I really tied to keep it withing 80 char width, not sure if such long conditional expressions are more readable.